### DISCUSSION TITLE
Firebase Crashlytics non-fatal issue in main.jsbundle

### DISCUSSION THREAD

**Discussion by TommyLeong** [invertase/react-native-firebase#4833] [Q&A] [answered]
Hello all, I've been receiving the following in my email recently. However, the same crash report can't be found on the portal itself when I go into my app's crashlytics details.

It only navigates me to this specific crash details while I tap on the `LEARN MORE` button in the email. Otherwise, I don't get to find it. Any idea what's this about?

I have the issue [posted on stackoverflow](https://stackoverflow.com/questions/65939473/firebase-crashlytics-non-fatal-issue-in-main-jsbundle)

<img width="353" alt="Screenshot 2021-01-29 at 2 24 23 PM" src="https://user-images.githubusercontent.com/9990179/106239503-b12d5f80-623d-11eb-9d47-7e4207fa9762.png">

**Comment by mikehardy** [comment, marked as answer]
Hi @TommyLeong 
This will be an upstream issue I think. If the report exists, but does not show up in the console by default, that seems to be a problem in the console or some sort of filtering settings in your specific project. I will be surprised if there is anything actionable in this module

**Reply by TommyLeong** [reply]
What I can assure is the filter is cleared. I even tried to filter out based on the specific app version, but there's no similar result showed.

What sort of upstream issue u mean? 🤔

**Reply by mikehardy** [reply]
I mean, I am just some person on the internet. I help maintain the open source react-native-firebase project, on behalf of Invertase who started it. Notice that "google" does not appear in that sentence :-). Google maintains firebase-android-sdk / firebase-ios-sdk with some community help but they own it. They are "upstream", they are official. The firebase web console is 100% google and is not on github, that is upstream. I have no ability to do anything with the web console, so this is something "upstream" and you'll have to take it up with google + the firebase web console team

**Reply by TommyLeong** [reply]
Yup Mike, I do know you're the contributor to RNFirebase. You have helped me a few times over our past discussion (:

I see. Thanks for directing me to the right direction!
